generator/typesystem_network.xml

```diff
@@ -5,10 +5,6 @@
   </namespace-type>
 
   <rejection class="QSqlError::Unused"/>
-  <rejection class="QSslConfiguration" function-name="defaultDtlsConfiguration"/>
-  <rejection class="QSslConfiguration" function-name="dtlsCookieVerificationEnabled"/>
-  <rejection class="QSslConfiguration" function-name="setDefaultDtlsConfiguration"/>
-  <rejection class="QSslConfiguration" function-name="setDtlsCookieVerificationEnabled"/>
```

Comment:
I think we can safely assume that nowadays Qt is always compiled with SSL support. If that really should be a problem, we may just add a respective tag attribute to be able to exclude these.